strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/providers/layout/AbstractLayoutProvider.java

```diff
@@ -465,8 +463,7 @@ protected void doDeletePath(RepositoryPath repositoryPath,
     {
         Files.delete(repositoryPath);
 
-        Repository repository = repositoryPath.getFileSystem()
-                                              .getRepository();
```

Comment:
@carlspring why you don't like such type of formatting? I think that it's more readable when invocation chain paced on new line.